packages/target-ethers-v5/src/index.ts

```diff
@@ -195,3 +197,39 @@ function createRootIndexContent(rootIndexes: FileDescription[], paths: string[])
 
   return [...rootIndexContent].join('\n')
 }
+
+function panicOnOldTypeScriptVersion() {
+  const requiredTSVersion = { major: 4, minor: 3 }
+  const targetEthersVersion = require('../package.json').version
+
+  let major: number
+  let minor: number
+
+  try {
+    const { version } = require('typescript')
+    ;[major, minor] = version.split('.').map(Number)
+
+    if (major > requiredTSVersion.major || (major === requiredTSVersion.major && minor >= requiredTSVersion.minor)) {
```

Comment:
tbh i would probably extract this to a pure, isolated function and test only that one but this is fine too ;)